### DISCUSSION TITLE
Automatically delete head branches via branch protection rule

### DISCUSSION THREAD

**Discussion by HonkingGoose** [facebook/docusaurus#5195] [Ideas]
This repository has a lot of stale branches, most of which have a PR that's been merged into production. So you don't need the branch anymore.

I propose you enable the repository setting: **Automatically delete head branches**:

> You can have head branches automatically deleted after pull requests are merged in your repository.

https://docs.github.com/en/github/administering-a-repository/configuring-pull-request-merges/managing-the-automatic-deletion-of-branches

Cleanup of the existing stale branches will need to be done manually.

**Comment by slorber** [comment]
Didn't know about this option, asked to get it enabled but Joel is in holiday so we may have to wait.
Will do the cleanup once it's turned on
